ppr-ui/src/composables/mhrInformation/useMhrUnitNote.ts

```diff
@@ -114,6 +114,17 @@ export const useMhrUnitNote = () => {
 
   const isCancelUnitNote = computed((): boolean => getMhrUnitNoteType.value === UnitNoteDocTypes.NOTE_CANCELLATION)
 
+  const isExpiryDatePassed = (note: UnitNoteIF): boolean => {
+    if (note.documentType === UnitNoteDocTypes.CONTINUED_NOTE_OF_CAUTION ||
```

Comment:
These two if conditions could be combined since both return false otherwise.

Reply:
Modified! Version updated as well!

Reply:
Do you really need two nested `if` conditions?

Reply:
Modified